ci-operator/step-registry/openshift/manifests/cgroupsv1/openshift-manifests-cgroupsv1-commands.sh

```diff
@@ -4,28 +4,11 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-cat >> "${SHARED_DIR}/manifest_mc-master-cgroupsv1.yml" << EOF
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
+cat >> "${SHARED_DIR}/manifest_mc-cgroupsv1.yml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Node
 metadata:
-  labels:
-    machineconfiguration.openshift.io/role: master
-  name: 99-master-cgroupsv1
+  name: cluster
 spec:
-  kernelArguments:
-    - systemd.unified_cgroup_hierarchy=0
-    - systemd.legacy_systemd_cgroup_controller=1
-EOF
-
-cat >> "${SHARED_DIR}/manifest_mc-worker-cgroupsv1.yml" << EOF
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 99-worker-cgroupsv1
-spec:
-  kernelArguments:
-    - systemd.unified_cgroup_hierarchy=0
-    - systemd.legacy_systemd_cgroup_controller=1
+  cgroupMode: "v1"
```

Comment:
I'm currently double checking whether this API is available in older versions. If not I'll have to leave the "v2" manifest as it was before that PR, because we have v2 jobs down to 4.10:
https://prow.ci.openshift.org/?job=*cgroupsv2*

However for v1, we're good since the jobs are only for 4.14 and beyond which I'm sure has this API: https://prow.ci.openshift.org/?job=*cgroupsv1*

Reply:
The `cgroupMode` API was added in 4.11, so I'll revert the change on the v2 manifest.

Reply:
Lets rename this step to cgroupv1-pre411 and create a new workflow using it? Later on we'll phase it out, but I think it would be useful to have it anyway